demo/admin/src/common/MasterMenu.tsx

```diff
@@ -55,9 +56,11 @@ export const masterMenuData: MasterMenuData = [
         },
     },
     {
+        type: "collapsible",
         primary: <FormattedMessage id="menu.pageTree" defaultMessage="Page tree" />,
         icon: <PageTree />,
         submenu: pageTreeCategories.map((category) => ({
```

Comment:
Now that we differentiate based on type, I'd name this `items`

Reply:
Renamed in 0b789f0d02101a66d88aba6b66c3241221672070